parkhangs-webapp/src/components/EventForm.js

```diff
@@ -18,13 +18,19 @@ class EventForm extends React.Component {
         super(props)
 
         this.state = {
+            eventTitle: this.props.eventTitle || null,
             eventDetail: this.props.eventDetails || null,
             eventStartTime: this.eventStartTime(),
             eventEndTime: this.eventEndTime(),
-            eventDate: this.dayPickerDateFormat(),
+            eventDate: this.eventDate(),
```

Comment:
I made this change in my branch as well

Reply:
yeah I added this after I asked you about it

Reply:
Could you just remove this one line? I’m going to merge in and I think aside from this there should be no merge conflicts ... or I’ll deal with it lol 

Everything looks good now